components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Edge.Publisher/tests/Storage/publishednodesschema.json

```diff
@@ -1,184 +0,0 @@
-{
```

Comment:
@suneetnangia - I removed the test folder copy of the schema and moved to a build output content copy approach. So linking the main schema file into the test project and setting copy always to the linked file so that we don't have to maintain two copies of the same file.